spec/features/idv/in_person_threatmetrix_spec.rb

```diff
@@ -0,0 +1,309 @@
+      complete_prepare_step(user)
+      complete_location_step
+      complete_state_id_step(user)
+
+      # ssn page
+      complete_ssn_step(user, tmx_status)
+      complete_verify_step(user)
+      complete_phone_step(user)
+      complete_enter_password_step(user)
+      acknowledge_and_confirm_personal_key
+
+      profile = InPersonEnrollment.last.profile
+      profile.deactivate_for_fraud_review
+      expect(profile.fraud_review_pending_at).to be_truthy
+      expect(profile.fraud_rejection_at).to eq(nil)
+      profile.update(fraud_review_pending_at: 31.days.ago)
+      FraudRejectionDailyJob.new.perform(Time.zone.now)
+
+      # profile is rejected
+      profile.reload
+      expect(profile.fraud_review_pending_at).to be(nil)
+      expect(profile.fraud_rejection_at).to be_truthy
+    end
+
+    context 'Fraud decisioning is completed' do
+      before do
+        complete_entire_ipp_flow(user, tmx_status)
+      end
+
+      context 'User passes IPP and passes TMX Review' do
+        it_behaves_like 'initially shows the user the barcode page'
+
+        it_behaves_like 'shows the user the Please Call screen'
+
+        it 'does not allow the user to restart the IPP flow', allow_browser_log: true do
+          deactivate_profile_update_enrollment(status: :passed)
+
+          visit_idp_from_sp_with_ial2(sp)
+          expect(page).to have_current_path(idv_please_call_path)
+          page.visit('/verify/welcome')
+          expect(page).to have_current_path(idv_please_call_path)
+          page.visit('/verify/in_person/document_capture')
+          expect(page).to have_current_path(idv_please_call_path)
+        end
+
+        it 'shows the user the successful verification screen after passing TMX review',
+           allow_browser_log: true do
+          deactivate_profile_update_enrollment(status: :passed)
+          visit_idp_from_sp_with_ial2(sp)
+          expect(page).to have_current_path(idv_please_call_path)
+
+          review_pass.run(args: [user.uuid], config:)
+          page.visit('/verify/welcome')
+          expect(page).to have_current_path(idv_activated_path)
+        end
+      end
+
+      context 'User passes IPP and fails TMX Review' do
+        it_behaves_like 'initially shows the user the barcode page'
+
+        it_behaves_like 'shows the user the Please Call screen'
+
+        it 'does not allow the user to restart the flow', allow_browser_log: true do
+          deactivate_profile_update_enrollment(status: :passed)
+
+          # user revisits before fraud rejection
+          visit_idp_from_sp_with_ial2(sp)
+          expect(page).to have_current_path(idv_please_call_path)
+
+          # reject the user
+          review_reject.run(args: [user.uuid], config:)
+
+          # user revisits after fraud rejection
+          visit_idp_from_sp_with_ial2(sp)
+          expect(page).to have_current_path(idv_not_verified_path)
+        end
+      end
+
+      # To be completed in a future ticket
```

Comment:
[Ticket added](https://cm-jira.usa.gov/browse/LG-12729) to address these 2 commented out specs